app/src/main/java/com/xmartlabs/taskloans/ui/screens/signin/SignInFragment.kt

```diff
@@ -15,47 +15,47 @@ import kotlin.time.ExperimentalTime
  * Created by mirland on 25/04/20.
  */
 class SignInFragment : BaseViewBindingFragment<FragmentSigninBinding>() {
-  private val viewModel: SignInFragmentViewModel by viewModel()
+    private val viewModel: SignInFragmentViewModel by viewModel()
```

Comment:
the indentation is wrong, please copy the style files from gong https://github.com/xmartlabs/gong/tree/master/.idea